test/intl402/NumberFormat/prototype/format/format-rounding-increment-10.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-intl.numberformat.prototype.format
+description: When set to `10`, roundingIncrement is correctly applied
+features: [Intl.NumberFormat-v3]
+includes: [testIntl.js]
+---*/
+
+var locales = [
+  new Intl.NumberFormat().resolvedOptions().locale, 'ar', 'de', 'th', 'ja'
+];
+var numberingSystems = ['arab', 'latn', 'thai', 'hanidec'];
+
+testNumberFormat(
+  locales,
+  numberingSystems,
+  {roundingIncrement: 10, maximumFractionDigits: 2},
+  {
+    '1.100': '1.1',
```

Comment:
As you noted, currently in the spec there is no difference between `{roundingIncrement:1, maximumFractionDigits:1}` and `{roundingIncrement:10, maximumFractionDigits:2}`. This is likely to change after we resolve the following issue:

https://github.com/tc39/proposal-intl-numberformat-v3/issues/81

In the mean time, we should move forward with these tests as written, and then update the tests after that issue has been resolved.